Mathlib/GroupTheory/SpecificGroups/Alternating.lean

```diff
@@ -362,6 +520,146 @@ instance isSimpleGroup_five : IsSimpleGroup (alternatingGroup (Fin 5)) :=
+        convert Multiset.toFinset_card_le {last m, σ (last m), j, σ j} using 2
+        simp
+      _ ≤ ({last m, σ (last m), j, σ j}ᶜ : Finset (Fin (m + 1))).card := by
+        rw [Finset.card_compl]
+    simp_rw [Finset.le_card_iff_exists_subset_card, Finset.card_eq_two] at hxy
+    obtain ⟨_, hxys, x, y, hnxy, rfl⟩ := hxy
+    have hcnd : (↑[j, x, y] : Cycle (Fin (m + 1))).Nodup := by
+      rw [Finset.subset_compl_comm, Finset.subset_iff] at hxys; simp at hxys ⊢; tauto
+    have hcnt : (↑[j, x, y] : Cycle (Fin (m + 1))).Nontrivial := by
+      rw [Cycle.nontrivial_coe_nodup_iff hcnd]; simp
+    let c := Cycle.formPerm (↑[j, x, y] : Cycle (Fin (m + 1))) hcnd
+    have ecs : c.support = {j, x, y} := by
+      rw [Cycle.support_formPerm _ hcnd hcnt, Cycle.coe_toFinset]; simp
+    have hcma : c ∈ alternatingGroup (Fin (m + 1)) := by
+      rw [mem_alternatingGroup, Cycle.isCycle_formPerm _ hcnd hcnt |>.sign,
+        Cycle.support_formPerm _ hcnd hcnt, Cycle.coe_toFinset, ← List.toFinset_eq hcnd]; rfl
+    use c * σ * c⁻¹
+    use by
+      simp_rw [normal_subgroupOf_iff_le_normalizer hHa, SetLike.le_def,
+        mem_alternatingGroup, mem_normalizer_iff] at hHn
+      specialize @hHn c hcma σ
+      exact hHn.mp hσH
+    use by
+      have hlnmcs : last m ∉ (c⁻¹).support := by
+        simp_rw [support_inv, ecs]
+        simp [Finset.subset_iff] at hjl hxys ⊢; tauto
+      have hσlnmcs : σ (last m) ∉ c.support := by
+        simp [ecs, Finset.subset_iff] at hjl hxys ⊢; tauto
+      simp [not_mem_support.mp hlnmcs, not_mem_support.mp hσlnmcs]
+    rw [DFunLike.ne_iff]
+    use j
+    have ecij : c⁻¹ j = y := by
+      simp_rw [c, ← Cycle.formPerm_reverse, Cycle.reverse_coe,
+        show [j, x, y].reverse = [y, x, j] from rfl,
+        Cycle.formPerm_apply_mem_eq_next (↑[y, x, j])
+          (Cycle.nodup_reverse_iff.mpr hcnd) j (by simp)]
+      simp only [Cycle.next, Cycle.ofList, Quot.hrecOn, Quot.recOn, Quot.rec]
+      apply List.next_getLast_cons <;> [simp; skip; rfl; skip] <;>
+        simp [Finset.subset_iff] at hxys ⊢ <;> tauto
+    simp_rw [Perm.mul_apply, ecij]
+    by_cases hσy : σ y ∈ c.support
+    case neg =>
+      rw [not_mem_support.mp hσy, σ.injective.ne_iff]
+      simp [Finset.subset_iff] at hxys; tauto
+    case pos =>
+      intro ecσy; rw [← apply_mem_support, ecσy, ecs] at hσy
+      revert hσy; simp [Finset.subset_iff] at hxys ⊢; tauto
+
+theorem isSimpleGroup_of_five_le_card (h5 : 5 ≤ card α) : IsSimpleGroup ↥(alternatingGroup α) := by
+  haveI : IsSimpleGroup ↥(alternatingGroup (Fin (card α))) := isSimpleGroup_of_five_le h5
+  obtain ⟨e⟩ := Fintype.truncEquivFin α
+  have me : ↥(alternatingGroup α) ≃* ↥(alternatingGroup (Fin (card α))) :=
+    e.permCongrMulEquiv.subgroupMap (alternatingGroup α) |>.trans <|
+      MulEquiv.subgroupCongr <| by
+        ext1 p
+        simp [e.permCongr.symm.surjective.exists,
+          show sign (e.permCongr.symm p) = sign p by simp, - permCongr_symm]
```

Comment:
Do you think it worth to extract as a lemma that the image of `alternatingGroup` under `permCongrMulEquiv` is also `alternatingGroup` and `permCongrMulEquiv` induces an isomorphism between `alternatingGroup` (maybe you can use `Subgroup.equivMapOfInjective` to define such isomorphism)?